blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -36,7 +36,7 @@ export function pipelineBranchesUnsupported(pipeline) {
 
 const classicConfigLink = (pipeline) => {
     let link = null;
-    if (AppConfig.getInitialUser() !== 'anonymous') {
+    if (AppConfig.getInitialUser().id !== 'anonymous') {
```

Comment:
Perhaps we could standardize on using the Security.isAnonymousUser() here? Realize you didn't write this code, but having "anonymous" magic string isn't good IMO.